src/components/features/MessageBox/MessageContent/index.tsx

```diff
@@ -6,12 +6,23 @@ import { MessageContentProps } from './MessageContent.type';
 import * as dayjs from 'dayjs';
 import 'dayjs/locale/ko';
 import MessageCheckBox from '../MessageCheckBox';
+import { useNavigate } from 'react-router-dom';
 
 const MessageContent = ({ message, checkMode, checkMessages, onToggleCheckMessage }: MessageContentProps) => {
 	dayjs.locale('ko');
 
+	const navigate = useNavigate();
+
+	const moveToMessageDetail = (messageId: number) => {
+		if (checkMode) {
+			return;
+		} else {
+			navigate(`/message/${messageId}`);
+		}
+	};
```

Comment:
이건 코드 스타일 차이이긴 한데 `if (...) return ;` 을 하셨으면 `else` 로 분기를 나누지 않아도 괜찮을 것 같습니다!
반영은 하지 않으셔도 괜찮습니다~

```suggestion
	const moveToMessageDetail = (messageId: number) => {
		if (checkMode) return;
		
		navigate(`/message/${messageId}`);
	};
```